pymatgen/analysis/transition_state.py

```diff
@@ -239,4 +264,35 @@ def from_dir(cls, root_dir, relaxation_dirs=None):
             else:
                 outcars.append(Outcar(outcar[0]))
                 structures.append(Poscar.from_file(contcar[0]).structure)
-        return NEBAnalysis(outcars, structures)
+        return NEBAnalysis.from_outcars(outcars, structures)
+
+    def as_dict(self):
```

Comment:
as and from_dict does not seem to be required since default implementation in MSONable seems to work just fine?